src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -64,20 +63,35 @@ public class SarifLogger : IDisposable, IAnalysisLogger
                 }
             }
 
-            string invocation = Environment.CommandLine;
+
+            run.Invocation = Invocation.Create();
 
             if (invocationTokensToRedact != null)
             {
-                foreach (string tokenToRedact in invocationTokensToRedact)
+                run.Invocation.Machine = Redact(run.Invocation.Machine, invocationTokensToRedact);
```

Comment:
just a 'secure by default' placeholder until we can drop in wiring to make this option. I was interested in seeing the redaction behavior. instead of this, I will simply make a bool value for emitting more sensitive information and make it false by default.